### PR TITLE
Fix bad merge

### PR DESCRIPTION
**Pull request by TonyBrobston**

### CODE REVIEW

File: src/main/java/org/dsmhack/controller/ProjectController.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public Project getProjectById(@PathVariable String projectId) {
     public ResponseEntity<CheckIn> checkUserIn(@PathVariable String projectId, @RequestBody String userGuid){
         CheckIn checkIn = new CheckIn();
         checkIn.setUserGuid(userGuid);
-        checkIn.setProjGuid(projectGuid);
+        checkIn.setProjGuid(projectId);
```
Comment (AndreasChristianson) [inline src/main/java/org/dsmhack/controller/ProjectController.java]: Should this variable be renamed to `projectGuid` ?

File: src/main/java/org/dsmhack/config/AppUserDetailsService.java [hunk 2]
```diff
@@ -22,10 +22,10 @@
     @Override
     public UserDetails loadUserByUsername(String userGuid) throws UsernameNotFoundException {
         LoginToken securityCode = loginTokenRepository.findByUserGuid(userGuid);
-        boolean isTokenExpired = securityCode.getTokenExpDate().before(Timestamp.valueOf(LocalDateTime.now()));
+        boolean credentialsNonExpired = LocalDateTime.now().isBefore(securityCode.getTokenExpDate());
```
Comment (AndreasChristianson) [inline src/main/java/org/dsmhack/config/AppUserDetailsService.java]: LocalDateTime includes timezone info (I think). Would it be less error prone to work in utc or gmt?

File: src/main/java/org/dsmhack/controller/ProjectController.java (hunk 1 above)
Comment (TonyBrobston) [reply src/main/java/org/dsmhack/controller/ProjectController.java]: Yeah, I had fixed all the naming awhile back; but I ended up reverting that commit, lol...

File: src/main/java/org/dsmhack/config/AppUserDetailsService.java (hunk 2 above)
Comment (TonyBrobston) [reply src/main/java/org/dsmhack/config/AppUserDetailsService.java]: At some point we should probably pick a time done that matters. On that note, I wonder what it defaults to.

**Pull request merged**